### ISSUE TITLE
Support for multiple folders

### ISSUE THREAD

**Issue by ChildishGiant** [owncloud/music#760]
I'd love if I was able to have multiple folders scanned since I have some external drives with music on.

**Comment by JoshuaPettus** [comment]
Out of curiosity what if you were to put a softlink within the main folder at the OS level?  Wouldn't it be treated as another subfolder within nextcloud/owncloud and music?   Just an idea for time being.

**Comment by coamtaker618** [comment]
@JoshuaPettus my motivation for wanting this nextcloud app to support multiple folders is that the Android app that I use as the client allows you to select a subset of all folders to be available as the library on that device.  This means that I can select music when I want it, and more importantly deselect music when I do not want it.  This also makes it easier to allow multiple users to each have their own music without needing to consolidate and de-duplicate audio on the server for users in that situation.

**Comment by JoshuaPettus** [comment]
That makes sense. And a little different from what the OP was asking for. Bringing files that are outside of Nextcloud into Music.  In which case they have to be brought into Nextcloud first somehow, I would think.

**Comment by coamtaker618** [comment]
@JoshuaPettus yes, at least in my case, all music is indeed brought into Nextcloud normally.  And that's true it is a little different than what OP is asking for...but I hope not TOO much of a thread hijack!!

**Comment by paulijar** [comment]
Kamino closed and cloned this issue to [nc-music/music](https://github.com/nc-music/music/issues/36)
